superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx

```diff
@@ -490,6 +491,10 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
     contentType === 'chart' &&
     (isFeatureEnabled(FeatureFlag.ALERTS_ATTACH_REPORTS) || isReport);
 
+  const dashboardFormatOptionEnabled =
+    contentType === 'dashboard' &&
+    (isFeatureEnabled(FeatureFlag.ALERTS_ATTACH_REPORTS) || isReport);
+
```

Comment:
If we really want to introduce a new variable `dashboardFormatOptionEnabled` we really should rename the old one on line 490 to `chartFormatOptionEnabled`.

Reply:
Thank you! updated code.